src/driver/postgres/PostgresDriver.ts

```diff
@@ -622,30 +622,34 @@ export class PostgresDriver implements Driver {
         if (!parameters || !Object.keys(parameters).length)
             return [sql, builtParameters];
 
+        const queryPlaceholders: ObjectLiteral = {};
+        Object.keys(nativeParameters).map((key, index) => { queryPlaceholders[key] = "$" + index + 1; });
```

Comment:
Wouldn't this be incorrect because of how order of operations works?

Let's say `index` is 4.

```
let index = 4;
console.log("$" + index + 1);
```

This would emit `$41`